### PR TITLE
Fixes #3234. ProcessContinuousButtonPressedAsync is always passing the start location.

### LINKED ISSUE #3234: ProcessContinuousButtonPressedAsync is always passing the start location.
**Describe the bug**
It really has been a tremendous struggle for scrolling to work properly. Now I discovered the cause of its malfunction by verifying that the location used is always the same as at the beginning, while the mouse button is pressed and moved simultaneously.

**To Reproduce**
1. Go to 'Scrolling' scenario
2. Click on 'left tee' and 
3. Scroll to right with the mouse button pressed
4. See in the mouse coordinates at right that the location is displaying the same location sometimes

**Expected behavior**
The location must display the right location.

**Screenshots**

https://github.com/gui-cs/Terminal.Gui/assets/13117724/5c1aceaa-ad4e-4426-8c8e-7e9d119dd170

**Additional context**
This issue is affecting all the v2 drivers (`WindowsDriver`, `CursesDriver` and `NetDriver`) and in the v1 is affecting more the `WindowsDriver`, although the others drivers are also using the start location but the scrolling works smoothly.

### PR DESCRIPTION
**Pull request by BDisp**
## Fixes

- Fixes #3234

## Proposed Changes/Todos

- [x] WindowsDriver
- [x] EscSeqUtils used by CursesDriver and NetDriver.

## Pull Request checklist:

- [x] I've named my PR in the form of "Fixes #issue. Terse description."
- [x] My code follows the [style guidelines of Terminal.Gui](https://github.com/gui-cs/Terminal.Gui/blob/develop/.editorconfig) - if you use Visual Studio, hit `CTRL-K-D` to automatically reformat your files before committing.
- [x] My code follows the [Terminal.Gui library design guidelines](https://github.com/gui-cs/Terminal.Gui/blob/develop/CONTRIBUTING.md)
- [x] I ran `dotnet test` before commit
- [ ] I have made corresponding changes to the API documentation (using `///` style comments)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any poor grammar or misspellings
- [x] I conducted basic QA to assure all features are working

### CODE REVIEW

**Comment by BDisp** [comment]
@tig do you want I do a R# cleanup code on this PR?

File: Terminal.Gui/ConsoleDrivers/EscSeqUtils/EscSeqUtils.cs [hunk 1]
```diff
@@ -1085,17 +1088,17 @@ private static async Task ProcessContinuousButtonPressedAsync (MouseFlags mouseF
 	{
 		while (isButtonPressed) {
 			await Task.Delay (100);
-			//var me = new MouseEvent () {
-			//	X = point.X,
-			//	Y = point.Y,
-			//	Flags = mouseFlag
-			//};
+			var me = new MouseEvent () {
```
Comment (tig) [inline Terminal.Gui/ConsoleDrivers/EscSeqUtils/EscSeqUtils.cs:1091]: `me` does not appear to be referenced below. What am I missing?

**Review by tig — APPROVED** [review]

**Comment by tig** [comment]
> @tig do you want I do a R# cleanup code on this PR?

No. Once you address my review comment we can merge this.

File: Terminal.Gui/ConsoleDrivers/EscSeqUtils/EscSeqUtils.cs (hunk 1 above)
Comment (BDisp) [reply Terminal.Gui/ConsoleDrivers/EscSeqUtils/EscSeqUtils.cs:1091]: Before it was using always the same location passed on the parameter which is called when the flag is only the button pressed without the the report mouse move. When the mouse with the button pressed is moving the task method isn't called and thus the passed parameter hasn't the current location and the `move_point` field need to be used.

Comment (tig) [reply Terminal.Gui/ConsoleDrivers/EscSeqUtils/EscSeqUtils.cs:1091]: I don't see where `me` is referenced. Can you show me?

Comment (BDisp) [reply Terminal.Gui/ConsoleDrivers/EscSeqUtils/EscSeqUtils.cs:1091]: I see your point, sorry. It really is not used. I'll fix that. The fixed was on updated the point variable.

Comment (BDisp) [reply Terminal.Gui/ConsoleDrivers/EscSeqUtils/EscSeqUtils.cs]: Done. Thanks.

**Review by tig — APPROVED** [review]

**Pull request merged**